src/PRo3D.Core/Transformation-Model.fs

```diff
@@ -285,6 +300,7 @@ module Transformations =
                 scaling              = scaling
                 trafoChanged         = false
                 usePivot             = false
+                pivotSize            = Initial.initPivotSize 4.0
```

Comment:
it might be a good idea to factor out readPivotSize which is then used top level where version dispatching occurs. What do you think?